packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -106,9 +105,9 @@ const PLANS = [
   },
 ];
 const SUBSCRIPTION_ID_1 = 'sub-8675309';
-const PAYMENT_TOKEN_VALID = '8675309-foobarbaz';
+//const PAYMENT_TOKEN_VALID = '8675309-foobarbaz';
 const PAYMENT_TOKEN_NEW = 'new-8675309';
-const PAYMENT_TOKEN_BAD = 'thisisabadtoken';
+//const PAYMENT_TOKEN_BAD = 'thisisabadtoken';
```

Comment:
Guessing this is area is under construction 🚧 but be sure to delete these commented-out constants before landing :+1: